pyiron_base/storage/parameters.py

```diff
@@ -936,17 +943,19 @@ def _lines_to_dict(self, lines):
                 lst["Comment"].append("")
         return lst
 
-    def _type_to_hdf(self, hdf):
+    def _type_to_dict(self):
```

Comment:
So this is one of the reasons I keep insisting on a common mixin class for to_dict/from_dict.  If you had this it could be predefined automatically, instead you had to add this now manually here, in #1260 and in #1250 (#1248 should have also had it).

Reply:
(now that it's done, go ahead with this now anyway, just making my point.)

Reply:
@pmrv I added an interface class, maybe you can have another look tomorrow.